wordpress-mobile/WordPress-Android
# Issue/13326 init history view pager

Parent issue #13326

This PR
- Adds ScanHistoryListFragment: An instance per view pager tab will be used, ScanHistoryFragment will serve as parent of the ViewPager. I decided not to put the ViewPager into the Activity in order to keep the activity as simple as possible in case we decide to move to a single activity structure.
- Moves Toolbar from ScanHistory Activity to ScanHistoryFragment: Same reasons as above
- Adds ViewPager2 to ScanHistoryFragment
- Adds "lift on scroll" (Z-axis) behavior. I tried adding show/hide on scroll behavior but it didn't work, I decided not to spent more time on it right now and add it later. 

To test:
1. Make sure Scan feature flag is enabled
2. Make sure a site with Scan feature is selected
3. Open Scan
4. Click on "History" action in the toolbar
5. Notice a viewpager with 3 tabs (3 ScanHistoryListFragments) is shown.
6. Scroll down and notice the toolbar is lifted as you start scrolling (z-axis)

PR submission checklist:
